package.json

```diff
@@ -52,9 +49,12 @@
     "yargs-parser": "^21.0.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.17.0",
     "@babel/eslint-parser": "^7.17.0",
     "@babel/plugin-proposal-class-properties": "^7.0.0",
```

Comment:
AFAIK, `@babel/plugin-proposal-class-properties` is now part of `@babel/preset-env` (since a long time), so I see no reason for Encore to manage it directly.

Reply:
btw, this would probably also solve https://github.com/symfony/webpack-encore/issues/1019

Reply:
Changed for `@babel/plugin-proposal-partial-application` instead.